### ISSUE TITLE
Humio.java isn't in the requirements.yml

### ISSUE THREAD

**Issue by jamesmckibbenathrb** [humio/ansible-demo#40]
The requirements.yml doesn't specify a 'humio.java' version to update to, and, it seems ansible when you process the requirements.yml doesn't grab the dependencies.

It seems the current dependency for humio.java is '0.6.0' and I had '0.5.0' cached.

So I edited the requirements.yml to have this addition:

`- src: humio.java`
`  version: 0.6.0`

Afterwards, I ran:
`ansible-galaxy install -r requirements.yml --force`

It was then that the package was cached and ready for deployment/use.

Please add this and update as necessary, thank you!
#39

**Comment by srdo** [comment]
I think this is fixed by changing our documentation to recommend `--force-with-deps` instead of `--force`.

See https://docs.ansible.com/ansible/latest/cli/ansible-galaxy.html#cmdoption-ansible-galaxy-collection-install-force-with-deps

Installing humio.java 0.5.0, then doing `--force-with-deps` on the bare_metal demo will upgrade the local humio.java version to 0.6.0.

**Comment by srdo** [comment]
@schofield What do you think?

**Comment by schofield** [comment]
@srdo We do mention it in the README for bare_metal, but we may need to update some other documentation to make this clearer. 

Since the other roles require humio.java I don't think we need to include it in the requirements for the this repo.

**Comment by srdo** [comment]
@schofield You're right. I think the docs were out of date though, see https://github.com/humio/docs.humio.com/pull/1027
